packages/sysutils/busybox/scripts/init

```diff
@@ -24,6 +24,7 @@
   /bin/busybox mkdir -p /dev
   /bin/busybox mkdir -p /proc
   /bin/busybox mkdir -p /sys
+  /bin/busybox mkdir -p /tmp
```

Comment:
It is - https://github.com/LibreELEC/LibreELEC.tv/pull/44/files#diff-708fce2b98cae7109f84bb7011142f00R635

I could have changed it, but I thought having /tmp around might prove useful in future...